src/tsconfig/rulesets/configurable-options.ts

```diff
@@ -18,5 +18,8 @@ configurableOptions.shouldPass('noUnusedParameters', Match.ANY);
 configurableOptions.shouldPass('resolveJsonModule', Match.ANY);
 configurableOptions.shouldPass('experimentalDecorators', Match.ANY);
 configurableOptions.shouldPass('noFallthroughCasesInSwitch', Match.ANY);
+configurableOptions.shouldPass('verbatimModuleSyntax', Match.ANY);
```

Comment:
Should we have a test for this too?

Reply:
I'm not sure. These are technically all tested by [this proptest](https://github.com/aws/jsii-compiler/pull/1262/files#diff-30fef5b012f460ebc57986fabb93bb19f85cb44c785343fe4b5596823983f3e1R38). What the test doesn't protect against is typos, which is the only reason the `outDir` test got added (a regression). WDYT?

Reply:
Hmm. Why did you add a specific test for `isolatedDeclarations` and `isolatedModules` then?

Reply:
I realized it afterwards and forgot to delete them 🙈  I'll take them out.